components/gpio_control/GPIODevices/simple_button.py

```diff
@@ -169,19 +173,24 @@ def __del__(self):
 
     @property
     def is_pressed(self):
+        # TODO should this be 'if pull_up_down == GPIO.PUD_UP'? pull_up is always true!
```

Comment:
Does anyone knows more about this? Seems like in the past "pull_up" was replaced by "pull_up_down". 
Has this been forgotten and is a bug?